### DISCUSSION TITLE
Migration worked perfectly

### DISCUSSION THREAD

**Discussion by ralfkrause** [AlmaLinux/almalinux-deploy#24] [General]
I just migrated one of my systems from CentOS 8.3 to AlmaLinux 8.3 without any issues.
The migrated system is an old HP MicroServer Gen 8 with Celeron processor.

The complete process took about 30 minutes.
So far verything seems to work perfectly.

Thank you so much guys.

**Comment by ExpertWebWorld** [comment]
dear sir,

we using centos  v8.5 with cpanel

so now if we use to update from centos to almalinux using command line then its 100% sure all of cpanel account not effect . AFter deploy all will work as same as previous. 

if you have step by step command please tell us
